### PR TITLE
Project navigation

### PR DESCRIPTION
**Pull request by rudolfs**

### CODE REVIEW

File: app/src/pages/Project/Source.svelte [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<script>
+  import { link } from "svelte-spa-router";
```
Comment (xla) [inline app/src/pages/Project/Source.svelte]:
```suggestion
```

File: app/src/components/ProjectSidebarItem.svelte [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<style>
+  .icon {
+    margin-right: 12px;
+    align-self: center;
+  }
+</style>
+
+{#if active}
+  <div class="icon">
+    <svelte:component this={icon} style="fill: var(--color-purple)" />
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/components/ProjectSidebarItem.svelte]: Hm, I'm not so sure. An icon could be used in different contexts where an active state could be different in each case.

File: app/src/components/ProjectSidebar.svelte [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    width: 250px;
+    height: 100%;
+    background-color: var(--color-almostwhite);
+    display: flex;
+    position: fixed;
+    left: 68px;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 44px 20px 0 24px;
+    z-index: 900;
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/components/ProjectSidebar.svelte]: Good catch, was using it to test things.

File: app/src/components/ProjectSidebarItem.svelte (hunk 2 above)
[the remark this answers is not in the document]
Comment (xla) [reply app/src/components/ProjectSidebarItem.svelte]: Yeah was more thiking out loud. As we discussed out-of-band, manipulating styles, classes directly seems brittle to me as it reveals too much of the shape in the sub-component/element and might break in unforeseen ways. Nothing really we can do here nad now about it tho.

File: app/src/pages/Project/Overview.svelte [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <script>
   import { link } from "svelte-spa-router";
-  import Layout from "../components/Layout.svelte";
+  import Layout from "../../components/Layout.svelte";
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/pages/Project/Overview.svelte]: Agree, would be nice to have.

I tried to get https://github.com/dot-build/rollup-plugin-includepaths to work for 1/2 hour, but got stuck with: "Uncaught Error: Function called outside component initialization" caused by `setClient(client);` in App.svelte. Giving up for now as it's out of scope for this PR.

[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/pages/Project/Overview.svelte]: You can give it a try if you feel lucky, I got this far: https://github.com/oscoin/mvp/commit/1836a16a4ad367a965f73a129f042354d06aa807

File: app/src/components/ProjectSidebar.svelte [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    overflow-y: scroll;
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/components/ProjectSidebar.svelte]: Should be fixed now.

File: app/src/components/ProjectSidebar.svelte [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+  nav :global(li:hover svg) {
+    fill: var(--color-purple);
+  }
+</style>
+
+<nav>
+  <ul>
+    <li>
+      <a href="/projects/{projectId}/overview" use:link>
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/components/ProjectSidebar.svelte]: Yeah, feel the same way. Have a look now!

**Pull request merged**